### DISCUSSION TITLE
[探索]实体语料标注工具yedda

### DISCUSSION THREAD

**Discussion by bifenglin** [X-lab2017/open-wonderland#37] [2️⃣ 聊项目]
当我们针对一大段文本需要标注里面的是实体语料，想要构建点边关系，构建一定的图模型时候或者需要提取里面的实体内容，可以使用yedda工具。相关地址：
[github仓库](https://github.com/jiesutd/YEDDA)
[论文地址](https://aclanthology.org/P18-4006.pdf)
[博客说明](https://chowdera.com/2022/04/202204210606143893.html)

**Comment by will-ww** [comment]
> 当我们针对一大段文本需要标注里面的是实体语料，想要构建点边关系，构建一定的图模型时候或者需要提取里面的实体内容，可以使用yedda工具。相关地址： [github仓库](https://github.com/jiesutd/YEDDA) [论文地址](https://aclanthology.org/P18-4006.pdf) [博客说明](https://chowdera.com/2022/04/202204210606143893.html)

感觉还是挺有用的，正好和今天Frank提出的几个点子可以关联上，做NLP的分析和标注~ 
- https://github.com/X-lab2017/open-research/issues/55
- https://github.com/X-lab2017/open-research/issues/56
